### PR TITLE
Comments popup

### LINKED ISSUE #14: [4pt]  -Display Comments pop up with selected item's details - Student B
Create a comments popup only with the top section (displaying details of the selected item)

Screenshot 2021-06-27 at 22 10 45

When the popup loads, the webapp retrieves data from the selected API and shows details about the selected item.

Add event to button prepared by your teammate that will open your popup. You need to communicate with your team member to make sure that you do not block each other (most likely the easiest solution is to merge the list of items feature to the dev branch before you try to add that event).

### PR DESCRIPTION
**Pull request by idungstanley**
* Create a comments popup only with the top section (displaying details of the selected item)
*  Add event to button prepared by your teammate that will open your popup

### CODE REVIEW

File: src/modules/comment.js [hunk 1]
```diff
@@ -1,16 +1,32 @@
...
 export default class Comment {
   static createCommentPopUp = (data) => {
     const newArticle = document.createElement('article');
     newArticle.innerHTML = `<div class= "flex"> 
-  <img src= ${data.url} alt="Image"/> 
+  <img src= ${data.artworkUrl100} alt="Image"/> 
   <span class='material-symbols-outlined'>close</span>
   </div>
-  <h2>Space ${data.count}</h2>
+  <h2>Space ${data.index}</h2>
```
Comment (wandergithub) [inline src/modules/comment.js]: - [x] I think we could use the name of the album here.  
image

File: src/modules/comment.js [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
   <div class= "pop-details">
     <p>Fuel titatium</p>
     <p>Length: 10000</p>
     <p>Weight: 10000</p>
     <p>Power: 10000</p>
   </div>`;
```
Comment (wandergithub) [inline src/modules/comment.js]: - [x] One of the [requirements ](https://github.com/idungstanley/API-based-webAPP/issues/14#issue-1237792105) for this task is to display specific information about the card. I think we could use the ID to request particular information and then display it. Example: Album number of tracks, collection Price, country, primary gender.

**Review by wandergithub — CHANGES_REQUESTED** [review]
## Changes Required: ♻️

- We are required to display personalized information on each card. Checkout detailed comments.
- Besides that, everything looks good.

File: src/modules/comment.js (hunk 2 above)
Comment (idungstanley) [reply src/modules/comment.js]: Done

File: src/index.js [hunk 3]
```diff
@@ -9,12 +10,14 @@ const displayItems = async (artistId = '271256') => {
     `https://itunes.apple.com/lookup?id=${artistId}&entity=album&limit=6`,
   );
   const obj = await response.json();
+  console.log(obj.results);
```
Comment (wandergithub) [inline src/index.js:13]: Let's remove all `console.log` entries before merging.

**Review by wandergithub — APPROVED** [review]
✔️ Approved, All card details looking good.

**Pull request merged**